src/Window.cpp

```diff
@@ -146,52 +145,50 @@ void Window::handle_launch()
 
 void Window::handle_login()
 {
-  this->_loginWidgetUI.btn_login->setDisabled(true); // probably a valid mutex?
+  this->_loginWidgetUI.btn_login->setDisabled(true); // lock mutex (probably safe?)
```

Comment:
This is not a mutex. It just prevents the button click event from firing twice. Whatever the implementation, the comment is unecessary.